FrostyEditor/ViewModels/Windows/ProfileSelectWindowViewModel.cs

```diff
@@ -62,9 +68,25 @@ public ProfileSelectWindowViewModel()
     [RelayCommand]
     private async Task AddProfile()
     {
+        FilePickerFileType Exe = new("Executable")
```

Comment:
since v2 supports not only windows versions of the game, i would just not have any type filter here

Reply:
xbox and ps?

Reply:
> [Avalonia](https://avaloniaui.net/) is a cross-platform UI framework for dotnet, providing a flexible styling system and supporting a wide range of platforms such as Windows, macOS, Linux, iOS, Android and WebAssembly.

`macOS` only has [Sims 4](https://www.ea.com/games/the-sims/the-sims-4)(not frostbite engine)
There's no any frostbite game that running on `Linux`, `iOS`, `Android` or a `Web browser`
Avalonia not support `Nintendo Switch` also need crack
Avalonia not support `PlayStation`
Xbox might work but it doesn't look compatible